dbux-code/src/traceDetailsView/nodes/StaticTraceTDNodes.js

```diff
@@ -49,14 +49,14 @@ const groupByMode = {
     for (const trace of traces) {
       const { contextId } = trace;
       const context = app.dataProvider.collections.executionContexts.getById(contextId);
-      const { parentTraceId } = context;
+      const parentTraceId = context.parentTraceId || 0;
       if (!tracesByParent[parentTraceId]) tracesByParent[parentTraceId] = [];
       tracesByParent[parentTraceId].push(trace);
     }
     const groups = tracesByParent
       .map((children, parentTraceId) => {
         const trace = app.dataProvider.collections.traces.getById(parentTraceId);
-        const label = makeTraceLabel(trace) || 'No parent';
+        const label = trace ? makeTraceLabel(trace) : 'No Parent';
```

Comment:
suggestion: change text to `'(no parent)'` (makes it clearer to the eye, I think)

Reply:
@MiccWan